privaterelay/tests/views_tests.py

```diff
@@ -545,3 +545,16 @@ def test_version_view(client, version_json_path) -> None:
     version_json_path.write_text(json.dumps(version_info))
     response = client.get("/__version__")
     assert response.json() == version_info
+
+
+@pytest.mark.django_db
+def test_heartbeat_view(client) -> None:
+    response = client.get("/__heartbeat__")
+    assert response.status_code in (200, 500)
```

Comment:
question (non-blocking): why is `status_code=500` okay here?

Reply:
It's one of the two possible responses... Let's be bold and say it is 200 and see where it fails.